std_msgs/Header header
convex_decomposer/Polyhedron[] polyhedra
